app/Helpers/DropzoneHelper.php

```
@@ -122,9 +122,8 @@ public static function DropzoneScript($url_key = null, $client_id = 1, $site_url
         // Allow extentions system
         $content_types = [];
         if ($acceptedExts === null) {
-            // Default
-            $CI = & get_instance();
-            $CI->load->model('upload/mdl_uploads');
+            // Default // TODO: Replace with Laravel patterns
+            // TODO: Use dependency injection - 'upload/mdl_uploads');
             $content_types = array_keys($CI->mdl_uploads->content_types);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$CI'. (undefined)

(UnusedLocalVariable)